### PR TITLE
Updating the batch files for 'perf.groovy'; SDK:release/2.0.0

### PR DESCRIPTION
**Pull request by ghost**
Updating the batch files for 'perf.groovy'

**Test perf CI run:**
https://ci2.dot.net/job/dotnet_sdk/job/perf/job/release_2.0.0/job/SDK_Perf_Windows_NT_x64/17/

**Results at BenchView:**
https://benchview/trendline?build_selector=latest&count=20&aggregate=arithmeticMean&filterTail=one&filterVal=100&interval=INTERVAL_MIN_MAX&rtids=[1084]&archids=[9]&mpids=[1292]&cfgids=[2689]&testids=[110118,110119,110121,110122,110124,110125]&jobgroup=SDK%20Perf%20Tests&jobtype=rolling&branchId=468&

@DrewScoggins

### CODE REVIEW

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

File: perf.groovy [hunk 1]
```diff
@@ -39,46 +39,24 @@ def static getBuildJobName(def configuration, def os) {
...
-                   batchFile("py \"${perfWorkingDirectory}\\Microsoft.BenchView.JSONFormat\\tools\\upload.py \"${perfWorkingDirectory}\\submission.json\" --container coreclr")
+                   batchFile("\"%WORKSPACE%\\build.cmd\" -configuration ${configuration} -sign -ci -perf /p:PerfIterations=10 /p:PerfOutputDirectory=\"${perfWorkingDirectory}\" /p:PerfCollectionType=stopwatch")
+
+                   // Upload perf results to BenchView
+                   batchFile("set perfWorkingDirectory=${perfWorkingDirectory}\n" +
+                   "set configuration=${configuration}\n" +
+                   "set architecture=${architecture}\n" +
+                   "set runType=${runType}\n" +
+                   "set BenchviewCommitName=${BenchviewCommitName}\n" +
+                   "\"%WORKSPACE%\\build\\uploadperftobenchview.cmd\")
```
Comment (DrewScoggins) [inline perf.groovy]: Missing a closing quote here.

**Comment by DrewScoggins** [comment]
Once you get this working with the updated batch script will you be able to port this back to the master branch?

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

File: perf.groovy (hunk 1 above)
Comment (johnbeisner) [reply perf.groovy]: Yup, fixed...

**Comment by tannergooding** [comment]
> Once you get this working with the updated batch script will you be able to port this back to the master branch?

release/2.0.0 flows into master, so the change will make its way back. We are able to fast-track this process if required.

**Comment by johnbeisner** [comment]
@DrewScoggins 
release/2.0.0 regularly merges with master

**Review by tannergooding — APPROVED** [review]
I think there is still room for improvement, but this satisfies the biggest request I had and we can fixup anything else later

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

File: build/uploadperftobenchview.cmd [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+py "%perfWorkingDirectory%\Microsoft.BenchView.JSONFormat\tools\build.py" git --branch %GIT_BRANCH_WITHOUT_ORIGIN% --type "%runType%" -o "%perfWorkingDirectory%\build.json"
+
+echo Creating: "%perfWorkingDirectory%\machinedata.json"
+py "%perfWorkingDirectory%\Microsoft.BenchView.JSONFormat\tools\machinedata.py" -o "%perfWorkingDirectory%\machinedata.json"
+
+echo Creating: "%perfWorkingDirectory%\measurement.json"
+pushd "%perfWorkingDirectory%"
+for /f "tokens=*" %%a in ('dir /b/a-d *.xml') do (
+    echo Processing: "%%a"
+    "%perfWorkingDirectory%\Microsoft.BenchView.JSONFormat\tools\measurement.py" xunitscenario "%%a" --better desc --append -o "%perfWorkingDirectory%\measurement.json"
```
Comment (DrewScoggins) [inline build/uploadperftobenchview.cmd]: Need to use py to call the Python script

Comment (johnbeisner) [reply build/uploadperftobenchview.cmd]: Yup, Thanks

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

**Comment by johnbeisner** [comment]
@dotnet-bot test ci please

**Pull request merged**